### ISSUE TITLE
request for minimum number of answers 

### ISSUE THREAD

**Issue by JelleK4NL** [Jefferson49/joomla4_plugin_sexy_polling#31]
Trying to use Sexy Polling Reloaded for board elections. 
Voters must select 5 out of 9 'answers' but can currently vote with minimum of 1 selected.
Can this parameter be added please?

**Comment by Jefferson49** [comment]
Thank you for filing the issue! Let me ask some questions to better understand it:
- You have already activated "Multiple Answers" ="Yes"?
- You have set "Number Options" to 5?
- You want the users to provide a minimum of 5 votes in parallel? 
- Less than 5 parallel votes should be rejected?

**Comment by JelleK4NL** [comment]
Yes. Enabled "Multiple Answers".
Indeed Number options is 5.
User should select (in this case) 5 out of 9 options. 
With selecting a sixth, they will get a warning but not if they select less than 5.
Maybe a gray button as long as no 5 are selected.
Thanks! Jelle

**Comment by Jefferson49** [comment]
Thank you for providing the details! The idea behind the feature request is now very clear to me. 

While diving into the code and thinking about some options to implement this, I found out that it needs more changes than expected on first view: database layout, data model, backend code, front end code, front end JavaScript, front end message. Therefore, I tag it as an enhancement and keep it in the list for future extension of the plugin, but will not implement it on short term.

**Comment by JelleK4NL** [comment]
Hi Jefferson,
No problem! Thanks for looking into this.
Few other ideas for future releases:
- Multiple awnsers, the each awnser has its own colour check mark. Would rather just have 1 colour for all.
- Post voting message: being able to change the text the user get to see when casting his/her vote. 
- Somehow I was not able to use code in the text of the awnser. Like the sample content has, I could not do the same. Could have to do maybe with my Joomla installation.
KR
Jelle

**Comment by Jefferson49** [comment]
> Multiple answers, the each answer has its own colour check mark. Would rather just have 1 colour for all.

It needs some changes to introduce an additional option, but I consider it rather as a nice to have feature. 

> Post voting message: being able to change the text the user get to see when casting his/her vote.

Can you describe in more detail? I think after "regular" votes, there is no message. Do you want to change the text of warning messages if the vote was not accepted?

> Somehow I was not able to use code in the text of the awnser. Like the sample content has, I could not do the same. Could have to do maybe with my Joomla installation.

I can reproduce that it is not working. If entering some code like "\<tag\>text<\tag>", the tags are removed and only "text" is included in the answer shown. Surprisingly, this is already the case in the original plugin in Joomla 3. I will create a separate issue for that and will do further checks about the root cause.
